twig_server/database/Project.py

```diff
@@ -0,0 +1,64 @@
+from twig_server.database.connection import Neo4jConnection, Neo4jDriver
+from twig_server.database.native import Node, Relationship
+from twig_server.database.User import User
+from typing import Any, List, Optional
+from neo4j import Record
+
+
+class Project(Node):
+    _label_name = "Project"
+    _label_owner_relationship = "Project_Owner"
```

Comment:
Probably should be a constructor option in the superclass Node

Reply:
let's just leave it like that for now, i plan to refactor the code when i have a clearer idea of what needs to be done on the server side anyway